src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1161,6 +1163,14 @@ export default {
 			this.verificationSumbitted = true;
 			this.handleBasketValidation();
 		},
+		handleVerificationOptOut() {
+			this.showVerification = false;
+			this.showVerifyRemovePromoCredit = true;
+		},
+		handleCancelPromoOptOut() {
+			this.showVerifyRemovePromoCredit = false;
+			this.handleBasketValidation();
```

Comment:
Just to clarify - whether or not a lender decided to join or opt out of the promo fund's associated team, they'll still have to verify their email address at a later step, correct?

Reply:
Correct. If they decide to remain in the promo, the email verification will pop back up.